src/iwrapmessage.h

```diff
@@ -28,6 +28,7 @@ Contact: bt-trx.com, mail@bt-trx.com
 enum iWrapMessageType {
 	kEmpty,
 	kUnknown,
+	kSETTING_CALLSIGN,
```

Comment:
Please remove, SETTING_CALLSIGN is no iWrap message

Reply:
Solved with https://github.com/phl0/firmware/commit/9bece9392f208d0897e485a552e37f3b87ffc408. (Will rebase to 1.1.0 dev branch as soon as all issues are done).